ui/src/apollo/mutations.js

```diff
@@ -384,6 +419,17 @@ const deleteDomain = (apollo, domain) => {
   return response;
 };
 
+const withdraw = (apollo, uuid, organization) => {
```

Comment:
This mutation must also get the period of time to remove. Remember that an Individual could be affiliated to the same organization during different periods of time.